site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/widgets/Worker.ts

```diff
@@ -0,0 +1,103 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { DNode, WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+import { theme, ThemeableMixin } from '@dojo/widget-core/mixins/Themeable';
+import * as styles from '../styles/worker.css';
+
+export interface WorkerProperties extends WidgetProperties {
+	firstName?: string;
+	lastName?: string;
+	email?: string;
+	timePerTask?: number;
+	tasks?: string[];
+}
+
+const WorkerBase = ThemeableMixin(WidgetBase);
+
+@theme(styles)
+export default class Worker extends WorkerBase<WorkerProperties> {
+	private _isFlipped = false;
+
+	render(): DNode {
+		return v('div', {
+			classes: this.classes(styles.worker, this._isFlipped ? styles.reverse : null)
+		}, [
+			this._renderFront(),
+			this._renderBack()
+		]);
+	}
+
+	private _renderFront(): DNode {
+		const {
+			firstName = 'firstName',
+			lastName = 'lastName'
+		} = this.properties;
+
+		return v('div', {
+			key: 'front',
+			classes: this.classes(styles.workerFront),
+			onclick: this.flip
+		}, [
+			v('img', {
+				classes: this.classes(styles.image),
+					src: 'images/worker.svg' }),
+				v('div', [
+					v('strong', [ `${lastName}, ${firstName}` ])
+				])
+			]
+		);
+	}
+
+	private _renderBack(): DNode {
+		const {
+			firstName = 'firstName',
+			lastName = 'lastName',
+			email = 'unavailable',
+			timePerTask = 0,
+			tasks = []
+		} = this.properties;
+
+		return v('div', {
+			key: 'back',
+			classes: this.classes(styles.workerBack),
+			onclick: this.flip
+			}, [
```

Comment:
Do you mean the indentation? I'll commit a fix for that, but let me know if there's anything else you're referring to.